my_scripts/eval/eval_DAgRL_tf.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#SBATCH --job-name=ovon-eval
+#SBATCH --output=slurm_logs/eval/ovon-ver-%j.out
+#SBATCH --error=slurm_logs/eval/ovon-ver-%j.err
+#SBATCH --gpus a40:1
+#SBATCH --nodes 1
+#SBATCH --cpus-per-task 10
+#SBATCH --ntasks-per-node 1
+#SBATCH --partition=kira-lab,overcap
+#SBATCH --qos=short
+#SBATCH --signal=USR1@100
+
+
+export TORCH_DISTRIBUTED_DEBUG=DETAIL
+export GLOG_minloglevel=2
+export HABITAT_SIM_LOG=quiet
+export MAGNUM_LOG=quiet
+export MAIN_ADDR=$(scontrol show hostnames "${SLURM_JOB_NODELIST}" | head -n 1)
+
+MAIN_ADDR=$(scontrol show hostnames "${SLURM_JOB_NODELIST}" | head -n 1)
+export MAIN_ADDR
+JOB_ID=${SLURM_JOB_ID}
+
+source /nethome/asingh3064/flash/envs/etc/profile.d/conda.sh
+conda deactivate
+conda activate ovon
+
+split="val_seen"
+
+TENSORBOARD_DIR="tb/objectnav/train/"
+CHECKPOINT_DIR="data/checkpoints/"
+LOG_DIR="Logs/evalDaggerRL_${JOB_ID}.log"
+
+srun python -um ovon.run \
+  --run-type eval \
+  --exp-config config/experiments/transformer_rl_ft_sparse_pirl.yaml \
+  habitat_baselines.tensorboard_dir=${TENSORBOARD_DIR} \
+  habitat_baselines.eval_ckpt_path_dir=${CHECKPOINT_DIR} \
+  habitat_baselines.log_file=${LOG_DIR} \
+  habitat_baselines.checkpoint_folder=${CHECKPOINT_DIR} \
+  habitat.dataset.data_path=data/datasets/ovon/hm3d/v2/val_seen/val_seen.json.gz \
+  ~habitat.task.lab_sensors.objectgoal_sensor \
+  +habitat/task/lab_sensors@habitat.task.lab_sensors.clip_objectgoal_sensor=clip_objectgoal_sensor \
+  habitat.task.lab_sensors.clip_objectgoal_sensor.cache=siglip_seems_like_there_is_a_blank_ahead.pkl \
```

Comment:
Couldnt file this file in the current branch. Did you generate this yourself ? If so, how ?

Reply:
But do you have this file in your local branch or not ?

Reply:
yes i have the file

Reply:
Okay, is it provided in one of the download links in the repo README ?

Reply:
You got the name of the .pkl from the resume state config - but you actually have the file in your local repo so where did you get it from ? Did you generate it yourself or was it provided in one of the download links in the README ?